e2e/davinci-app/main.ts

```diff
@@ -81,13 +82,14 @@ const urlParams = new URLSearchParams(window.location.search);
 
 (async () => {
   const davinciClient: DavinciClient = await davinci({ config, logger, requestMiddleware });
-  const protectAPI = protect({ envId: '02fb4743-189a-4bc7-9d6c-a919edfe6447' });
+  const protectApi = protect({ envId: '02fb4743-189a-4bc7-9d6c-a919edfe6447' });
+  const fidoApi = fido();
```

Comment:
Does anything other than the `fido.js` component use this API? If no, could we just move this into the component, rather than initializing it here to just pass it into the component?